contracts/src/main/kotlin/com/template/contracts/AppointmentDateContract.kt

```diff
@@ -30,8 +30,8 @@ class AppointmentDateContract : Contract {
             "No inputs should be consumed when issuing a date" using (tx.inputs.isEmpty())
             "Only one output state is created" using (tx.outputs.size == 1)
 
-            val out = tx.outputStates[0] as AvailableAppointmentDate
-            "Dates must be of the format dd-MM-yyyy" using (checkDate(out.date))
+//            val out = tx.outputStates[0] as AvailableAppointmentDate
+//            "Dates must be of the format dd-MM-yyyy" using (checkDate(out.date))
```

Comment:
Why is it commented out?

Reply:
It was because the `tx.outputStates[0]` was a list of `TransactionStates` and not `StateAndRef`, I think I need to approve the date from the input states instead, but I wanted to fix the other issues before adding more to the PR :)